app/controllers/participations_controller.rb

```diff
@@ -6,10 +6,11 @@ def show
 
   def new
     @participating = @fleet.participating?(@eve_char_info.id)
+    @expired = @fleet.expired?

```

Comment:
Das würde ich tatsächlich nicht in eine Variable packen sondern einfach in der View ` @fleet.expired?` benutzen.